### ISSUE TITLE
Support ELBv2

### ISSUE THREAD

**Issue by lczerepuk** [bash-my-aws/bash-my-aws#250]
Hi,

My elbs command didnt show Application Load Balancers:

```
~/.bash-my-aws[master] $ elbs
~/.bash-my-aws[master] $
```

In source code I see 
```

elbs() {

  # List ELBs
  # Accepts LoadBalancer names or ARNS on STDIN and converts to LoadBalancer names
  #
  #     $ elbs
  #     elb-MyLoadBalancer-1FNISWJN0W6N9  2019-12-13T10:24:55.220Z  subnet-eff2cf88
  #     another-e-MyLoadBa-171CPCZF2E84T  2019-12-13T10:25:24.300Z  subnet-eff2cf88

  local elb_names=$(__bma_read_inputs | sed 's/arn[a-zA-Z0-9:\-]*loadbalancer\///g')
  local filters=$(__bma_read_filters $@)

  aws elb describe-load-balancers                                     \
    $([[ -n ${elb_names} ]] && echo --load-balancer-names $elb_names) \
    --query "LoadBalancerDescriptions[][
               LoadBalancerName,
               CreatedTime,
               join(',', Subnets)
             ]"          \
    --output text       |
  grep -E -- "$filters" |
  LC_ALL=C sort -k 2    |
  column -s$'\t' -t
}
```

Application/Network Load Balancers are managed by aws elbv2: https://docs.aws.amazon.com/cli/latest/reference/elbv2/index.html#cli-aws-elbv2

Can You fix it ? :)

**Comment by mbailey** [comment]
That would probably best be done by adding two new commands to list the new types of load balancer:

`elbs`: Classic Load Balancers (existing command)
`albs`: Application Load Balancers (v2)
`nlbs`: Network Load Balancers (v2)

**Comment by lczerepuk** [comment]
I prepared Pull Request for ALB/NLB support: https://github.com/bash-my-aws/bash-my-aws/pull/254

**Comment by mbailey** [comment]
Merged #254
